operators/pkg/controller/kibana/kibana_controller.go

```diff
@@ -110,14 +114,17 @@ var _ reconcile.Reconciler = &ReconcileKibana{}
 // ReconcileKibana reconciles a Kibana object
 type ReconcileKibana struct {
 	k8s.Client
-	scheme   *runtime.Scheme
-	recorder record.EventRecorder
+	operatorClient k8s.Client
```

Comment:
Why do we need the operatorClient in this struct?

Reply:
I forgot to remove it. :confused: